api/catalog_test.go

```diff
@@ -49,7 +49,7 @@ func TestAPI_CatalogNodes(t *testing.T) {
 					"wan": "127.0.0.1",
 				},
 				Meta:        map[string]string{},
-				CreateIndex: meta.LastIndex,
+				CreateIndex: meta.LastIndex - 1,
```

Comment:
The create index is not going to be deterministic given the number of things that can bump the index near startup (anti-entropy, unrelated changes to server-side state like the autopilot config, etc.). I think we might need a wrapper around `verify.Values` that looks for Raft info and performs sanity checks on the fields but doesn't directly compare them. We'd need to check that the index is always >0 and that the create index <= modify index; those are the invariants. Internally we always have a `RaftIndex` struct, though it looks like in the API structs some of these are flattened, so I'm not 100% sure what the best way to do this would be.

I can't think of a good way to make the create index deterministic, though I'll give it a little more thought.